bindings/rust/aws-lc-sys-template/src/lib.rs

```diff
@@ -1,6 +1,9 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0 OR ISC
 
+use paste::paste;
+use std::os::raw::{c_long, c_char, c_void};
```

Comment:
The bindings generated use primitive types from `std::os::raw` (not from `std::ffi`).